### ISSUE TITLE
CUDA.compilecache does not exists

### ISSUE THREAD

**Issue by 101001000** [JuliaParallel/rodinia#7]
In the nn benchmark, when the profile option is passed, CUDA.compilecache is called which is not defined in the CUDA package.

Maybe it was deprecated?

**Comment by maleadt** [comment]
Correct, that has been removed from CUDA.jl. It seems like some of the benchmarking functinality here has bitrotted, and it's probably best to remove it entirely (or at least the compilation measurements), as there's not really a public API to wipe the cache and measure compilation time.
